src/style.css

```
@@ -1 +1,61 @@
 /* add styles here */
+.page {
+  /* width: fit-content; */
+  margin: 0;
+
+  /* padding: 22px; */
+}
+
+.page__content {
+  margin: 0;
+}
+
+.stars {
+  /* width: fit-content; */
+
+  /* box-sizing: border-box; */
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars:not(:last-child) {
+  /* margin-bottom: 4px; */
+}
+
+/* #region pseudo */
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
```

```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--1 .stars__star:nth-child(-n + 1) {
  background-image: url(/src/images/star-active.svg);
}
```
combine all selectors with the comma

---

Please, check my work again